firmware/ventilator-controller-stm32/Core/Src/main.cpp

```diff
@@ -612,6 +620,7 @@ int main(void)
   // Normal loop
   while (true) {
     uint32_t current_time = hal_time.millis();
+    float abp_pressure = 0;
```

Comment:
`hfnc.sensor_vars()` will need to be modified to provide a place to store the measurement(s) produced by `abp.output(...)`. For now, maybe we can use `hfnc.sensor_vars().pout` (we'll probably need to rename this to a more standard name later).